modules/log.cpp

```diff
@@ -257,13 +258,19 @@ void CLogMod::PutLog(const CString& sLine,
     CString sLogDir = LogFile.GetDir();
     struct stat ModDirInfo;
     CFile::GetInfo(GetSavePath(), ModDirInfo);
-    if (!CFile::Exists(sLogDir)) CDir::MakeDir(sLogDir, ModDirInfo.st_mode);
-    if (LogFile.Open(O_WRONLY | O_APPEND | O_CREAT)) {
-        LogFile.Write(CUtils::FormatTime(curtime, m_sTimestamp,
-                                         GetUser()->GetTimezone()) +
-                      " " + (m_bSanitize ? sLine.StripControls_n() : sLine) +
-                      "\n");
-    } else
+    if (!CFile::Exists(sLogDir))
+        CDir::MakeDir(sLogDir, ModDirInfo.st_mode);
+
+    if (LogFile.Open(O_WRONLY | O_APPEND | O_CREAT))
+    {
+        /* dirty hack */
```

Comment:
What are the options here:
* Keep files open until end of day
* Write uncompressed logs, then compress them at midnight
* Anything else?

That's assuming %d is presented in file name... What to do if it's not?